scss/_utilities.scss

```diff
@@ -39,6 +39,66 @@ $utilities: map-merge(
       property: position,
       values: static relative absolute fixed sticky
     ),
+    "top": (
```

Comment:
Would we want to be more prescriptive with these utilities? Instead of all four directions, we do all four _corners_?

For example, `.top-right` vs `.top-100.right-0`?

Rest of those could include `.top-center`, `.top-left`, `.bottom-left`, `.bottom-center`, and `.bottom-right`. Those would fit the use case shown in the PR here, but I don't want to be too limiting for folks.

Reply:
Maybe, I've explored two possible options to simplify the utilities.

First of all, let's say we decide to use the following classes:

- ```.top-left```
- ```.top-center```
- ```.top-right```
- ```.center-left```
- ```.center-center```
- ```.center-right```
- ```.bottom-left```
- ```.bottom-center```
- ```.bottom-right```

## Using left and top allways

All utilities should use left and top to position the child, so ```.bottom-right``` would be: 
```css
.bottom-right {
  left: 100%;
  top: 100%;
}
```

#### Pros
- We only need one class to center them. Let's call it ```.translate-center```
```css
.translate-centered{
  transform: translateX(-50%) translateY(-50%);
}
```

#### Cons
- ```.bottom-right``` is not **bottom: 0** **right: 0** but **top: 100%** **left: 100%**

- The element cannot be absolute positioned *inside* the parent. This is not possible:
<img width="116" alt="Screenshot 2020-08-12 at 19 56 24" src="https://user-images.githubusercontent.com/385551/90051028-5330ae00-dcd7-11ea-878e-1b8a7e26b30b.png">

## Use class selectors to apply the correct props

For each utility, use the named props, so:
```css
.top-left {
  top:0; 
  left:0;
}
.bottom-right {
  bottom: 0; 
  right:0;
}
.center-right {
  top: 50%; 
  right: 0;
}
```

Then, using class selectors we can define the translation for each option:

```css
.top-left.translate-center {
    transform: translateX(-50%) translateY(-50%);
}
.bottom-right.translate-center {
    transform: translateX(50%) translateY(50%);
}
.center-right.translate-center {
    transform: translateX(50%) translateY(-50%);
}
```

### Pros
- Utility classes are named with the prop names (easy to understand)
- We only need one class to center them. ```.translate-center```

### Cons
- Requires more css selectors
- The element cannot be absolute positioned "outside" the parent
<img width="110" alt="Screenshot 2020-08-12 at 19 56 16" src="https://user-images.githubusercontent.com/385551/90051394-eb2e9780-dcd7-11ea-8508-1c78ace895df.png">


Let's open the debate 🙂...